<launch>
	<include file="$(find stage_navigation)/launch/laser_amcl.launch">
		<arg name="map_file" value="$(find assignment6)/maps/laser_world_map_version03.yaml" />
    </include>
    
	<node
		pkg="assignment6"
		type="simple_navigation_goals"
		name="navigator"
		launch-prefix="xterm -e"
	/>
	<param name="poseX1" value="-1.0" /> 
	<param name="poseY1" value="-1.0" />
	
	<param name="poseX2" value="-2.0" /> 
	<param name="poseY2" value="-2.0" />
	
	<param name="poseX3" value="1.0" /> 
	<param name="poseY3" value="1.0" />
	
	<param name="poseX4" value="2.0" /> 
	<param name="poseY4" value="-1.0" />
</launch>
